sortingUtils/sort.spec.js

```diff
@@ -0,0 +1,8 @@
+const sort = require("./sort");
+const assert = require("assert");
+
+describe("Sort", function() {
+  it("should sort sample array", function() {
+    assert.deepEqual([1, 6, 32, "a", "asd"], sort([6, "a", 1, "asd", 32]));
```

Comment:
deepEqual is already deprecated, please use deepStrictEqual

Reply:
Changed.